CONTRIBUTING.md

```diff
@@ -13,6 +13,9 @@ There are a few basic ground-rules for contributors:
 1. **External API changes and significant modifications** ought to be subject to an **internal pull-request** to solicit feedback from other contributors.
 1. Internal pull-requests to solicit feedback are *encouraged* for any other non-trivial contribution but left to the discretion of the contributor.
 1. Contributors should attempt to adhere to the prevailing code-style.
```

Comment:
A badge is not always the best to define code-style. Link to docs or standardjs ?

Reply:
we are using standardjs

Reply:
I mean we should write it down.

Reply:
Agreed.

Reply:
That's outside of the scope of this PR.

Reply:
> That's outside of the scope of this PR.

Agree.

Don't forget that if you press on the "code style standard" badge in our README.md you will be redirected to the standardjs website :)